### PR TITLE
Eason/119 small UI fix

### PR DESCRIPTION
**Pull request by WYC401**
## Description

1. Adjust font size and logo size
2. change tab "Join the Team" to "Join our team" in project page. Also, the order of tabs is changed/

## Updates

As I did before, I will copy and paste package.json and package-lock.json file from main branch before merging

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by WYC401** [comment]
[small ui fixes](https://trello.com/c/HyVRJNHt/119-small-ui-fixes)

**Review by svolterra — COMMENTED** [review]
LGTM! Nice work Eason! 👏  Just left a couple of comments + please do not include the `package.json` file in your PR though :)

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "yup": "^0.32.9"
   },
   "scripts": {
-    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-app-rewired start",
```
[the remark this answers is not in the document]
Comment (WYC401) [reply package.json]: has revert it back.
also, for the package-lock.json I copied and pasted the same file in main branch to revert my change back.

File: src/statics/projectNav.ts [hunk 2]
```diff
@@ -2,22 +2,23 @@ import ROUTES from "./routes";
...
+		REF: ROUTES.TEAM
+	},
 	{
 		TITLE: TEXT.PROJECT_NAV.PROJECT_DESCRIPTION,
 		REF: ROUTES.PROJECT_DESCRIPTION
 	},
 	{
-		TITLE: TEXT.PROJECT_NAV.JOIN_THE_TEAM,
+		TITLE: TEXT.PROJECT_NAV.JOIN_OUR_TEAM,
 		REF: ROUTES.JOIN_THE_TEAM
```
[the remark this answers is not in the document]
Comment (WYC401) [reply src/statics/projectNav.ts]: Has changed JOIN_THE_TEAM in the router

File: src/components/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -139,9 +139,9 @@ const Navbar: React.FC<{}> = () => {
       <div className="navbar-menu">
         <AppBar position="sticky" className="nav-appbar">
         <Toolbar className="nav-toolbar">
-          <div className="logo-container">
-            <img src={VCLIcon} alt="VCL logo" />
-            <p>{TEXT.COMMON.TITLE}</p>
+          <div className="logo-container" style={{width: "28%"}}>
```
Comment (ElisaHu1) [inline src/components/Navbar/Navbar.tsx:142]: image
the left side of the logo is a bit hidden

Comment (WYC401) [reply src/components/Navbar/Navbar.tsx:142]: Good observation! But I find the picture itself is not complete with its left part haha

**Review by ElisaHu1 — APPROVED** [review]
i am good with this, as long as you revert package.json

**Comment by WYC401** [comment]
> i am good with this, as long as you revert package.json

Have reverted!

**Pull request merged**